lost-paws/lost-paws
# Render a container component that contains our google map

## Overview

**Issue Type**


**Description**
Created our `MapContainer` component along with some styling to render on our landing page. 

**Ticket Item**
[Render a container component to render the map.](https://trello.com/c/R6ZNYFIQ/31-render-a-container-component-to-render-the-map)


**Steps to Validate Feature **
1. Go to localhost:5000 and see the container with a blue background for visibility.
2. Notice corners are round
3. Width and height take a large portion of the front page



**Previous behavior**
Blank white page

**Expected behavior**
Landing page should have a clear and visible light blue container with rounded edges.
